KotlinSample/src/main/res/values/strings.xml

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+    <string name="hello">This sample demonstrates the In Session type, which denotes that the survey is presented at the point where the user accepts the invitation. Follow the instructions below to check eligibility.</string>
+    <string name="reset_button_desc">Once the invite is shown, the SDK drops into an idle state until the repeat days have elapsed. Click here to reset the state of the SDK.</string>
```

Comment:
Maybe instead of "Click here to reset the state of the SDK" it should read "Use this button to reset the state of the SDK."

"Click here" seems a bit out of place.

Reply:
Cool. Thanks for the suggestion! 👍